internal/interpreter/dynamic_item_tree.rs

```diff
@@ -1372,7 +1384,7 @@ pub(crate) fn generate_item_tree<'id>(
         type_loader: std::cell::OnceCell::new(),
         #[cfg(feature = "internal-highlight")]
         raw_type_loader: std::cell::OnceCell::new(),
-        debug_handler: std::cell::RefCell::new(Box::new(|_, text| {
+        debug_handler: std::cell::RefCell::new(Rc::new(|_, text| {
```

Comment:
I mentioned this before, but this allocates for every component only to get replace later.
It would be better to have debug_handler as an `Option` and handle `None` specially in the debug hook.

Reply:
I am aware of that. On the plus side it avoids a branch whenever `debug(...)` is called.

Reply:
(A branch cost virtually nothing compared to a allocation.)

Reply:
Yes, but the allocation happens once after compilation, when the UI is frozen anyway.

`debug(...)` happens a lot and at a time users are interacting with the UI.